webhook_server_container/utils/github_repository_settings.py

```diff
@@ -186,6 +214,7 @@ def set_repositories_settings(config_: Config, github_api: Github) -> None:
     futures = []
     with ThreadPoolExecutor() as executor:
         for _, data in config_data["repositories"].items():
+            data = get_repo_branch_protection_rules(config_data=config_data, repo_data=data)
```

Comment:
please rename `data` to name that describe the data (`repo_branch_protection_rules`?)

Reply:
But this is the entire data dict including repo branch protection. Do you want me to just separate it?